src/Compilers/VisualBasic/Portable/Scanner/Scanner.vb

```diff
@@ -1469,7 +1469,7 @@ FullWidthRepeat:
         End Function
 
         Private Function TokenOfStringCached(spelling As String) As SyntaxKind
-            If spelling.Length = 1 OrElse spelling.Length > 16 Then
+            If spelling.Length > 16 Then
```

Comment:
I had hoped that there was a PR from my earlier work, but I think Vlad and I must have reviewed it offline.